meson.build

```diff
@@ -452,13 +452,13 @@ compiler_supports_int128 = compiler.links('''
 
 compiler_supports_float128 = compiler.links('''
 	#ifndef __SIZEOF_FLOAT128__
-		#error __SIZEOF_FLOAT128__ wasn't defined!
+		#error __SIZEOF_FLOAT128__ wasn t defined!
 	#endif
 	#ifndef __FLT128_MANT_DIG__
-		#error __FLT128_MANT_DIG__ wasn't defined!
+		#error __FLT128_MANT_DIG__ wasn t defined!
 	#endif
 	#ifndef __LDBL_MANT_DIG__
-		#error __LDBL_MANT_DIG__ wasn't defined!
+		#error __LDBL_MANT_DIG__ wasn t defined!
```

Comment:
Here I removed the `'` because they messed up my syntax highlighting, but I'll obviously put them back